### ISSUE TITLE
BatchNorm2d of ResNet model warning

### ISSUE THREAD

**Issue by liuyugeng** [SsnL/dataset-distillation#42]
Hello, when I use ResNet18 with pretrained model from pytorch. It will show the warning. 
`[WARNING]  BatchNorm2d contains buffer running_var. The buffer will be treated as a constant and assumed not to change during gradient steps. If this assumption is violated (e.g., BatchNorm*d's running_mean/var), the computation will be incorrect.` 
I am not sure if it will influence the results.

Many thanks

**Comment by SsnL** [comment]
If the BN layers are in evaluation mode, then it is fine. Otherwise it will be incorrect.

**Comment by liuyugeng** [comment]
That means if I use resnet18 to distill the images, it will be always incorrect coz it uses batchnorm when training. So the buffer will not become None forever.
Am I right about that?

**Comment by liuyugeng** [comment]
So, why the buffer can't change when the training phase? I can't get it in the paper.

**Comment by SsnL** [comment]
it is just not implemented in this repo. you are feel to implement it yourself.
